src/event-handler/remove-listener.ts

```diff
@@ -1,8 +1,8 @@
-import { State } from "../types";
+import { EmitterEventName, State } from "../types";
 
 export function removeListener(
   state: State,
-  webhookNameOrNames: string | string[],
+  webhookNameOrNames: EmitterEventName | EmitterEventName[],
```

Comment:
Will this work with `*` and `error`? Related: https://github.com/octokit/webhooks.js/pull/416#pullrequestreview-579913781

Reply:
Yup! `EmitterEventName` is the `keyof EmitterEventMap`, which is of type `EmitterWebhookEventMap & { error: WebhookEventHandlerError; };`

![image](https://user-images.githubusercontent.com/3151613/106394780-3d6c9c00-6463-11eb-9d9a-302121152d4e.png)